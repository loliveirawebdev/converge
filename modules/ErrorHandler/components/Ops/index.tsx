export default function Ops() {
  return null;
}
