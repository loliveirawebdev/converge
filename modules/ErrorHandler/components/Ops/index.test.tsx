import Ops from "./index";
import React from "react";
import { create } from "react-test-renderer";

test("if Ops screen still the same", async () => {
  const ops = await create(<Ops />);
  expect(ops).toMatchSnapshot();
});
