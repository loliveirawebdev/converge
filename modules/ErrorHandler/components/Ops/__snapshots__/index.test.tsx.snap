// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`if Ops screen still the same 1`] = `null`;
